src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleTable } from '../components/PeopleTable';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { getFilteredPeople } from '../utils/getFilteredPeople';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [searchParams] = useSearchParams();
+
+  const hasError = !isLoading && error;
+  const noPeople = !isLoading && people.length === 0;
```

Comment:
The `noPeople` condition checks if `people.length === 0`, which might not be accurate if the filtering results in an empty list. Consider checking `filteredAndSortedPeople.length` instead to ensure the message reflects the filtered results.